seminar/urls.py

```
@@ -0,0 +1,39 @@
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+from django.urls import re_path
```

Ruff (0.8.2)

`django.urls.re_path` imported but unused

Remove unused import: `django.urls.re_path`

(F401)